### ISSUE TITLE
changes in scatter3d axis label specification?

### ISSUE THREAD

**Issue by BoundaryValueProblems** [sglyon/PlotlyJS.jl#189]
@sglyon , I'm using plotlyjs backend in Plots.jl. In a very simple scatter3d plot, I want to specify xlabel, ylabel, zlabel.  Through Plots.jl's ```scatter``` function, I could do this in January 2018 without any problem. I have a script/function to do some 3D scatter plot, and I have a pdf file generated in January 2018, where I could see the plotlyjs backend did correct axis labeling. Now, the same script does not render xlabel, ylabel, zlabel as I specified. Instead, it shows simply, ```x, y, z``` on the plot. I'm currently using PlotlyJS.jl version 0.10.2 (master) with Plots.jl 0.17.1+ (master). I want to locate where the problem is. Have you changed anything related to the axis labels since January 2018 so that the same Plots.jl function ```scatter(randn(10),randn(10),randn(10), xlabel="A", ylabel="B", zlabel="C")``` is mapped to an incorrect plotlyjs command? 
This problem was reported in https://github.com/JuliaPlots/Plots.jl/issues/1485, but I wanted to check with you here because it is very important to resolve this problem. Thanks a lot for your help!

**Comment by sglyon** [comment]
HI @BoundaryValueProblems I'm sorry but nothing on the PlotlyJS.jl side has changed for this. 

If you could show me the plotly json that is created from your plot command then I would be able to help you and the Plots.jl team debug

Thanks!

**Comment by BoundaryValueProblems** [comment]
@sglyon thanks for offering your help! Could you tell me how to create the plotly json from my plot command? I've never explicitly tried this before. Previously, I generated an html file from the result of my Plots.jl command via 
```
plt=scatter(randn(10),randn(10),randn(10),xlabel="A",ylabel="B",zlabel="C"); #This is a Plots.jl function.
PlotlyJS.savefig(plt.o, "test.html")
```
Thanks!

**Comment by sglyon** [comment]
After looking at the json generated by Plots.jl , I think the problem is that Plots is setting  properties on `layout.scene.(x|y|z)axis1` instead of `layout.scene.(x|y|z)axis` -- the addition of the `1` is messing things up.

**Comment by BoundaryValueProblems** [comment]
Thanks a lot, @sglyon ! I'll report this to the Plots.jl team!

**Comment by mkborregaard** [comment]
That happens here: https://github.com/JuliaPlots/Plots.jl/blob/master/src/backends/plotly.jl#L361-L363
It looks like it happens on purpose, for when there are more than one subplot. Did the plotly specification change? Or is it simply an old mistake in the code that should be deleted?

**Comment by sglyon** [comment]
I think maybe it changed. I know that's the correct behavior when you have multiple subplots. But for some reason when you have just one it does not like the `1` to be there

**Comment by mkborregaard** [comment]
Oh, that's tricky because a Plots series doesn't know how many subplots are in the total plot.

**Comment by mkborregaard** [comment]
I wonder if it would error if you always had both keys?

**Comment by sglyon** [comment]
Nope, it wouldn't error. I figured out that this was the problem by adding a copy of the version with the 1 to the spec under the key without the 1. 

If we ever plan to validate the plotly json then we'd have problems, but we don't now.
